django/core/validators.py

```diff
@@ -168,12 +168,17 @@ def validate_integer(value):
 
 @deconstructible
 class EmailValidator(object):
+    """
+    This validator doesn't aim for complete spec strictness. It tries its best
```

Comment:
Which spec is that? Is there an RFC to cite? We might omit this until the validator uses HTML5 validation.